internal/ast/ast.go

```diff
@@ -11,7 +11,7 @@ import (
 // it := n.Children()
 // for it.Next() {
 // 		it.Node()
-// }
+// }.
```

Comment:
Guessing those extra dots were added by an auto-fix of some sort? :D If so let's revert and disable that.

Reply:
Must be an autofixer complying with recommendation in Go's [list of code review comments](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences).

Reply:
indeed, the godot linter, which wants... dots at the end of comments